### PR TITLE
Feature/works about me

### PR DESCRIPTION
**Pull request by Albinpirela**
##Add the works-about-me branch 
#Add style to branch
Captura de pantalla (35)
Captura de pantalla (37)
Captura de pantalla (38)

### CODE REVIEW

**Review by GerardoGomezG — APPROVED** [review]
Well done bro!
You're ready to merge this branch into "develop"
thank you @GerardoGomezG , I really appreciate your review, but I need you to please make one last review because I made some changes after the approval

File: index.html
Comment (GerardoGomezG) [inline index.html]: Hey dude!
Can you explain a little what these lines of codes where for?

File: index.html
Comment (GerardoGomezG) [inline index.html]: What's the difference with previous deleted lines of code?

File: index.html
Comment (GerardoGomezG) [reply index.html]: Ok @Albinpirela
I've seen it's related to SVG images, so keep on moving man...
You're doing well!

File: index.html
Comment (GerardoGomezG) [reply index.html]: I got it!
No further comments about this

**Review by GerardoGomezG — APPROVED** [review]
Hey @Albinpirela 
I've just checked the comments I left and you couldn't see. I saw the code was for the SVG images, so I don't need you to explain it anymore.
Keep working like this, you've got skills!

File: index.html
Comment (Albinpirela) [reply index.html]: Hi, happy day! I hope you are very well, I'll explain, the difference is the color of the vector image from #ff8303 to #ffffff a small change at the last minute. 😄

File: index.html
Comment (Albinpirela) [reply index.html]: Hello, exactly, they are xml code 💻 I await the approval. thank you

**Review by GerardoGomezG — APPROVED** [review]
Max-width 👍🏼 @Albinpirela

**Comment by Albinpirela** [comment]
## 
Captura de pantalla (57)
Captura de pantalla (58)
Captura de pantalla (59)
Add the work-about-me branch into develop

**Pull request merged**
